teamcity/common_build_feature.go

```diff
@@ -0,0 +1,105 @@
+package teamcity
+
+import (
+	"encoding/json"
+)
+
+type CommonBuildFeature struct {
```

Comment:
What do you think of renaming this to simply, `BuildFeature` (and renaming the file & constructor funtion to match)? I think it's likely that the more specific features, like `FeatureCommitStatusPublisher` will eventually either be refactored to just wrap this more generic type, or removed altogether.

Reply:
This file should have an associated `_test.go` file.